packages/commonwealth/server/routes/threads/create_thread_poll_handler.ts

```diff
@@ -43,7 +43,8 @@ export const createThreadPollHandler = async (
     threadId: parseInt(threadId, 10),
     prompt,
     options,
-    customDuration: custom_duration,
+    customDuration:
+      custom_duration === 'Infinite' ? Infinity : parseInt(custom_duration),
```

Comment:
parsInt("0") will give 0. I think we should check for that as well.

Reply:
the ternary condition will default to 5 days if the custom duration is falsey:
```ts
  const ends_at =
    customDuration === Infinity
      ? null
      : customDuration
      ? moment().add(customDuration, 'days').toDate()
      : moment().add(5, 'days').toDate();
```